app/services/parser.rb

```diff
@@ -0,0 +1,67 @@
+require 'open-uri'
+
+class Parser
+  HEADLESS_RUN = true
+  DELAY_VAL    = 3
+  AVITO_SOURCE = "https://www.avito.ru/irkutsk/kvartiry/sdam/na_dlitelnyy_srok".freeze
+
+  def self.get_avito(count = 2)
+    if HEADLESS_RUN
+      headless = Headless.new
+      headless.start
+    end
+    
+    doc = Nokogiri::HTML(open(AVITO_SOURCE));
+    
+    doc.css('h3.title a').each_with_index do |ad, index|
+      break if ++index == count
+      delay if index != 0
+      browser = browser_init
+      item = {}
+      link = ad['href']
+      item_doc = Nokogiri::HTML(open(item_link_avito(link)));
+      item[:renter] = item_doc.css('#seller strong').text.gsub(/\w|\s/,'')
+      item[:description] = item_doc.css('div#desc_text').text
+      item[:price] = item_doc.css('.p_i_price').text.gsub(/\D/,'')
```

Comment:
Space missing after comma.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.